definitions-terms.md

```diff
@@ -0,0 +1,54 @@
+# Definitions and Terms
+
+A collection of definitions and terms used within this repository.
+
+* [Artifact Author](#artifact-author)
+* [Distribution Operator](#distribution-operator)
+* [Media Type](#media-type)
+* [OCI Image](#oci-image)
+* [Registry](#registry)
+* [Well Known Type](#well-known-type)
+* [YASS](#yass)
+
+## Artifact Author
+
+The owner of an artifact format. The [OCI Image Spec](https://github.com/opencontainers/image-spec/) is owned by the OCI working group.
+
+An artifact is defined to be unique by its `config.mediaType`.
+
+## Container Registry
+
+See [Registry](#registry)
+
+## Distribution Operator
+
+Vendors that implement and/or run the [OCI Distribution Spec](https://github.com/opencontainers/distribution-spec/).
+
+## Media Type
+
+The uniqueness of an artifact is defined by its type. An artifact has a type, which has a collection of layers.
+
+The Artifact is defined as unique by its `manifest.config.mediaType`. Layers are defined by their `layer.config.mediaType`.
```

Comment:
Can we have some clarity regarding the suggestion for using `+` vs. `.` at the end of a mediatype? For example, when to use `.json` vs. `+json`?